src/test/java/apoc/export/cypher/ExportCypherTest.java

```diff
@@ -331,4 +253,66 @@ public void testExportQueryCypherFormatUpdateStructure() throws Exception {
         TestUtil.testCall(db, "CALL apoc.export.cypher.schema({file},{exportConfig})", map("file", output.getAbsolutePath(), "exportConfig", Util.map("format", "cypher-shell")), (r) -> {});
         assertEquals(EXPECTED_ONLY_SCHEMA_CYPHER_SHELL, readFile(new File(directory, "onlySchema.cypher")));
     }
+
+    @Test
+    public void testExportCypherNodePoint() throws FileNotFoundException {
+        db.execute("CREATE (f:Test {name:'foo'," +
+                "place2d:point({ longitude: 56.7, latitude: 12.78 })," +
```

Comment:
```suggestion
                "place2d:point({ x: 2.3, y: 4.5 })," +
```
le coordinate geospaziali le gestisci sotto, qui altrimenti hai gestito due volte WGS-2d